designer/server/lib/consoleMessages/publishError.ts

```diff
@@ -0,0 +1,19 @@
+import config from "../../config";
+
+function publishError() {
```

Comment:
perhaps there is too much documentation in logs now?

Reply:
Slimed down